framework/auth/views.py

```diff
@@ -246,12 +247,14 @@ def register_user(**kwargs):
             http.BAD_REQUEST,
             data=dict(message_long='Email addresses must match.')
         )
-    # TODO: Sanitize fields
     try:
+        full_name = request.json['fullName']
```

Comment:
No special reason for or against- I just find it more readable not to chain statements in this case.